### PR TITLE
Fix news

### PR DESCRIPTION
**Pull request by Easybuoy**
#### What does this PR do?
- Fix loading of news

#### Description of Task to be completed?
- Add cors proxy

#### How should this be manually tested?
- Visit a profile detail page

#### What are the relevant Trello Card stories? [ Trello Card Item ](https://trello.com/b/8x3mo2V5/say-their-names-web)
-

#### Pull Request Type (check one)
- [ ] Feature
- [x] Bug Fixes
- [ ] Chore

#### Developer Checklist
- [x] New code has been scanned for ESlint issues
- [x] I have tested my code locally
- [x] I made sure that there are no other pull requests that solve for this issue
- [x] My code follows the same structure within the guidelines

### CODE REVIEW

**Comment by ashleygraf101** [comment]
Change works on saytheirname/deploy-preview but not saytheirnames-dev/deploy-preview. Not sure if intentional.

**Comment by Easybuoy** [comment]
> rks on saytheirname/deploy-preview but not saytheirnames-dev/deploy-preview. Not sure if intentional.

I needed to add an environment variable to the dev environment before deploy, but the app automatically deployed when the pull request was raised. I'd re-deploy now

**Comment by Easybuoy** [comment]
> > rks on saytheirname/deploy-preview but not saytheirnames-dev/deploy-preview. Not sure if intentional.
> 
> I needed to add an environment variable to the dev environment before deploy, but the app automatically deployed when the pull request was raised. I'd re-deploy now

Fixed @ashleygraf101

File: .github/workflows/unit-test.yaml [hunk 1]
```diff
@@ -2,17 +2,21 @@ name: Unit Test Workflow
...
+        with:
+          node-version: 12
+      - name: Install Node Dependencies
+        run: yarn install --frozen-lockfile
+      - name: Run all Unit Tests
+        run: CI=true yarn test
+env:
+  REACT_APP_TWITTER_URL: https://twitter.com/SayTheirName_io
+  REACT_APP_FACEBOOK_URL: https://www.facebook.com/Say-Their-Names-108926117523025/?modal=admin_todo_tour
+  REACT_APP_INSTAGRAM_URL: Octohttps://www.instagram.com/saytheirname.iocat
```
Comment (samakintunde37) [inline .github/workflows/unit-test.yaml:22]: You might wanna check this. I believe it's supposed to be a link.

Comment (Easybuoy) [reply .github/workflows/unit-test.yaml:22]: Yeah, true

**Pull request merged**